docs/dictionary/command/mobileAddContact.lcdoc

```diff
@@ -22,37 +22,81 @@ mobileUpdateContact tContact
-- "prefix": The name prefix.
-- "suffix": The name suffix.
-- "organization": The name of the organization.
-- "jobtitle": The job title (iOS only).
-- "department": The name of the department (iOS only).
-- "message": A person message (iOS only).
-- "note": A person note.
+**Person Information** - The personal information of the contact is stored 
+at the top level of the array and has the following keys: 
 
+- `firstname`: The first name.
+- `middlename`: The middle name.
+- `lastname`: The last name.
+- `alternatename`: The alternative name.
+- `nickname`: The nick name (iOS only).
+- `firstnamephonetic`: The phonetic transcription of the first name.
+- `middlenamephonetic`: The phonetic transcription of the middle name.
+- `lastnamephonetic`: The phonetic transcription of the last name.
+- `prefix`: The name prefix.
+- `suffix`: The name suffix.
+- `organization`: The name of the organization.
+- `jobtitle`: The job title (iOS only).
+- `department`: The name of the department (iOS only).
+- `message`: A person message (iOS only).
+- `note`: A person note.
+
+**E-Mail Addresses** - The e-mail addresses of the contact are stored in 
+integer indexed arrays starting at one under the key email. This allows any 
+number of e-mail addresses to be stored against a particular category. 
+
+There are three categories of e-mail address:
+-	`home`: - The home e-mail address.       
+-	`work`: - The work e-mail address.
+-	`other`: - An alternative e-mail address.
+
+**Telephone Numbers** - The telephone numbers of the contact are stored in 
+integer indexed arrays starting at one under the key phone. This allows 
+any number of telephone numbers to be stored against a particular category. 
+
+There are ten categories of phone numbers:
+-	`mobile` - The mobile telephone number.       
+-	`iphone` - The iPhone telephone number (iOS only).
+-	`main` - The main telephone number (iOS only).
+-	`home` - The home telephone number.
+-	`work` - The work telephone number.
+-	`homefax` - The home FAX number (iOS only).
+-	`workfax` - The work FAX number (iOS only).
+-	`otherfax` - The other FAX number (iOS >= 5.0 only).
+-	`pager` - The pager number (iOS only).
+-	`other` - An alternative telephone number.
+
+**Address** - Addresses of the contact are stored as sub arrays under the 
+key address. 
+
+There are three categories of address:
+-	`home` - The home address.       
+-	`work` - The work address.
+-	`other` - The other address.
+
+**Address Subkey** - Each address category is an integer indexed array, 
+starting at one under the key address. This allows any number of 
+addresses to be stored against a particular category. 
+
+There are six categories of address subkey:
+-	`street` - The address street.       
+-	`city` - The address city.
+-	`state` - The address state.
+-	`zip` - The address ZIP code.
+-	`country` - The address country.
+-	`countrycode` - The address country code (iOS only).
 
```

Comment:
The best thing here I think would be to change the description of this parameter to something like `An array describing the contact.`, and move all of the rest of the description of it to the `Description:` section. As the param descriptions go in a table, they're not very conducive to very large structured descriptions.